src/main/java/org/jabref/gui/mergeentries/newmergedialog/ThreeWayMergeHeaderView.java

```diff
@@ -0,0 +1,52 @@
+package org.jabref.gui.mergeentries.newmergedialog;
+
+import javafx.geometry.Insets;
+import javafx.scene.control.Control;
+import javafx.scene.layout.GridPane;
+
+import org.jabref.gui.mergeentries.newmergedialog.cell.HeaderCell;
+import org.jabref.logic.l10n.Localization;
+
+/**
+ * GridPane was used instead of a Hbox because Hbox allocates more space for cells
+ * with longer text, but I wanted all cells to have the same width
+ */
+public class ThreeWayMergeHeaderView extends GridPane {
```

Comment:
Why do you inherit from the GridPane?  Favor composition over inheritance (FCoI principle) https://en.wikipedia.org/wiki/Composition_over_inheritance
Just have a field for the GridPane or create a method initGridPane() where you create a new GridPane and init your rows